backend/src/junior/dto/login.dto.ts

```diff
@@ -0,0 +1,11 @@
+import { IsNotEmpty, IsPhoneNumber } from 'class-validator';
+
+export class LoginJuniorDto {
+
+    @IsPhoneNumber(null)
```

Comment:
Talked about this earlier. I guess it fine leaving it as is but in the future we may want to change `@IsPhoneNumber(null)` to `@IsPhoneNumber('FI')`

Reply:
Will update now.